components/Main/Exercises/Exercises.js

```diff
@@ -3,10 +3,12 @@ import { StyledExercises } from "./StyledExercises";
 import React from "react";
 import Description from "./Description/Description";
 import { useState } from "react";
-import Image from "next/image";
+
+import SearchBar from "./SearchBar/SearchBar";
 
 export default function Exercises() {
   const [toggleIds, setToggleIds] = useState([]);
```

Comment:
maybe a nicer name would be "toggledExercises"? Because you don't only toggle the id but instead use the id to toggle the Details, correct?

Reply:
yes you are right :) im gonna change the name. Thank you!!